metadata.yml

```
@@ -1,20 +1,27 @@
 rules:
-- apiGroups: ["networking.k8s.io"]
-  apiVersions: ["v1"]
-  resources: ["ingresses"]
-  operations: ["CREATE", "UPDATE"]
+  - apiGroups:
+      - networking.k8s.io
+    apiVersions:
+      - v1
+    resources:
+      - ingresses
+    operations:
+      - CREATE
+      - UPDATE
 mutating: false
 contextAware: false
 annotations:
-  # artifacthub specific:
+  # artifacthub specific
   io.artifacthub.displayName: Ingress Policy
   io.artifacthub.resources: Ingress
   io.artifacthub.keywords: ingress
+  # kubewarden specific
   io.kubewarden.policy.ociUrl: ghcr.io/kubewarden/policies/ingress
-  # kubewarden specific:
   io.kubewarden.policy.title: ingress-policy
   io.kubewarden.policy.description: Enforce requirements on Ingress resources
   io.kubewarden.policy.author: Kubewarden developers <cncf-kubewarden-maintainers@lists.cncf.io>
   io.kubewarden.policy.url: https://github.com/kubewarden/ingress-policy
   io.kubewarden.policy.source: https://github.com/kubewarden/ingress-policy
   io.kubewarden.policy.license: Apache-2.0
+  io.kubewarden.policy.category: Ingress
+  io.kubewarden.policy.severity: medium
```

```suggestion
  io.kubewarden.policy.severity: high
```

For routes and ingress, maybe we can use high. Kyverno follows that approach.